### DISCUSSION TITLE
Hide table selection column

### DISCUSSION THREAD

**Discussion by Sedpower** [ant-design-blazor/ant-design-blazor#2480] [Q&A]
I would like to hide the column containing radio button that allow user to select a row. I tried with CSS but it just break the table layout.
![image](https://user-images.githubusercontent.com/83537027/170694826-496344b3-0e06-4295-91cb-e97e39585e21.png)
Do someone know how I could do that ?

**Comment by JesperOlsen** [comment]
That looks like one of the examples from the demo-website.
The Selection column is there because you have defined it in the coloumn-definition.
Remove this line:
`<Selection Key="@context.Name" Type="@selectionType" Disabled="@(context.Name == "Disabled User")" />`
